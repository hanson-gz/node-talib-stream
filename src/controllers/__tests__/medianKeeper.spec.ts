// import { MedianKeeper } from '../medianKeeper';
// import * as _ from 'lodash';
//
// import { sampleCandles } from './sampleCandles';
//
// describe('MedianKeeper', () => {
//   it('should work', () => {
//     const period = 10;
//     const close = _.map(sampleCandles, c => c.last).slice(0, 15);
//     const results: number[] = [];
//
//     const medianKeeper = new MedianKeeper({ period });
//     _.each(close, c => {
//       medianKeeper.add(c);
//       const keeperRes = medianKeeper.get();
//       results.push(medianKeeper.get());
//     });
//     expect(results).toMatchSnapshot();
//   });
// });
