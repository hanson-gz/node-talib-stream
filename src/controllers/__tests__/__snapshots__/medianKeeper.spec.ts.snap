// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MedianKeeper should work 1`] = `
Array [
  9305,
  9292.05,
  9292.05,
  9274.95,
  9274.95,
  9286.25,
  9286.25,
  9299.2,
  9295.8,
  9299.1,
  9299.1,
  9299.1,
  9299.1,
  9334,
  9370.2,
]
`;
